test/powershell/Modules/PackageManagement/PackageManagement.Tests.ps1

```diff
@@ -18,40 +18,44 @@ $InternalSource = 'OneGetTestSource'
 
 
 Describe "PackageManagement Acceptance Test" -Tags "Feature" {
- 
+
```

Comment:
In the future, please avoid making formatting changes in the same PR as code change per the [Coding Conventions](https://github.com/PowerShell/PowerShell/blob/master/docs/dev-process/coding-guidelines.md).